### DISCUSSION TITLE
Unique PayloadUUID for each profile

### DISCUSSION THREAD

**Discussion by miaLDN** [paulmillr/encrypted-dns#122] [General]
Would it be possible to have a unique PayloadUUID for each profile please?  A lot, but not all, share the ID listed below.  You can only have one profile installed where the IDs are the same.  It's useful to be able to select between a few DNS servers, occasionally some sites break with filtering rules as an example.

https://developer.apple.com/documentation/devicemanagement/configuring_multiple_devices_using_profiles

``` xml
<key>PayloadUUID</key>
	<string>A4475135-633A-4F15-A79B-BE15093DC97A</string>
<key>PayloadVersion</key>
```

**Comment by nasiralamreeki** [comment]
@paulmillr if you would allow me I can tackle this and submit a PR?

**Comment by paulmillr** [comment]
Yeah
